pyproject.toml

```diff
@@ -4,3 +4,11 @@ requires = ["setuptools", "wheel", "tableauhyperapi", "oldest-supported-numpy"]
 [tool.towncrier]
     package = "pantab"
     filename = "NEWS.rst"
+
+[tool.cibuildwheel]
+build = ["cp38-*", "cp39-*", "cp310-*", "cp311-*", "cp312-*"]
```

Comment:
It looks like pandas does not distribute 3.8 wheels anymore, so we can probably drop that here

Reply:
Done.

Reply:
Looks like there is still 38 here. Not sure if that is related to CI issue but worth removing